Formula/grpc.rb

```diff
@@ -32,9 +31,23 @@ class Grpc < Formula
   depends_on "protobuf"
   depends_on "re2"
 
+  uses_from_macos "llvm" => [:build, :test], since: :catalina
   uses_from_macos "zlib"
 
+  fails_with :clang do
+    build 1100
+  end
```

Comment:
Add a reason here please.

I've tested it out, it seems to be something to do with setting default values to struct fields while under C++17 mode. Switching to set the defaults in a constructor (requires a patch) or to use C++14 instead avoids the issue (though I understand C++17 is used because `abseil` is compiled that way).